addons/skin.estuary/xml/DialogPVRProviderManager.xml

```diff
@@ -0,0 +1,289 @@
+						<width>700</width>
+						<include>SettingsItemCommon</include>
+						<label>$LOCALIZE[19346]</label>
+					</control>
+					<control type="radiobutton" id="7">
+						<description>PVR Client Provider</description>
+						<width>700</width>
+						<include>SettingsItemCommon</include>
+						<label>$LOCALIZE[19345]</label>
+					</control>
+				</control>
+				<control type="image">
+					<left>600</left>
+					<top>215</top>
+					<width>70</width>
+					<height>50</height>
+					<aspectratio>keep</aspectratio>
+					<texture background="true">$INFO[Container(20).ListItem.Property(icon)]</texture>
+				</control>
+				<control type="grouplist">
+					<top>615</top>
+					<itemgap>-15</itemgap>
+					<onleft>60</onleft>
+					<onright>9000</onright>
+					<onup>14</onup>
+					<ondown>7</ondown>
+					<animation effect="slide" start="0,0" end="0,45" time="0" condition="!Control.IsVisible(31)">Conditional</animation>
+					<control type="label" id="9003">
+						<description>channel options Header</description>
+						<width>700</width>
+						<height>50</height>
+						<textoffsetx>30</textoffsetx>
+						<font>font12</font>
+						<label>$LOCALIZE[31021]</label>
+						<aligny>center</aligny>
+						<textcolor>button_focus</textcolor>
+						<shadowcolor>black</shadowcolor>
+					</control>
+					<control type="button" id="31">
+						<description>New provider Button</description>
+						<width>700</width>
+						<height>100</height>
+						<textoffsetx>40</textoffsetx>
+						<align>center</align>
+						<label>$LOCALIZE[19343]</label>
+					</control>
+					<control type="button" id="30">
+						<description>Channel Manager Button</description>
+						<width>700</width>
+						<height>100</height>
+						<textoffsetx>40</textoffsetx>
+						<align>center</align>
+						<label>$LOCALIZE[19347]</label>
+					</control>
+				</control>
+			</control>
+			<control type="grouplist" id="9000">
+				<right>20</right>
+				<top>90</top>
+				<onleft>9002</onleft>
+				<onright>20</onright>
+				<itemgap>dialogbuttons_itemgap</itemgap>
+				<width>400</width>
+				<include content="DefaultDialogButton">
+					<param name="width" value="400" />
+					<param name="id" value="4" />
+					<param name="label" value="$LOCALIZE[186]" />
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="width" value="400" />
+					<param name="id" value="5" />
+					<param name="label" value="$LOCALIZE[14070]" />
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="width" value="400" />
+					<param name="id" value="35" />
+					<param name="label" value="$LOCALIZE[19348]" />
+				</include>
+				<include content="DefaultDialogButton">
+					<param name="width" value="400" />
+					<param name="id" value="6" />
+					<param name="label" value="$LOCALIZE[222]" />
+				</include>
+				</control>
+				<control type="group">
+					<width>400</width>
+					<height>400</height>
+					<bottom>20</bottom>
```

Comment:
`<bottom>0</bottom>`

Reply:
So you mean with these changes it will be the same width as the buttons above it? Or something else?

Reply:
yes, same width as the buttons and bottom of the icon aligned to the listbox on the left
![center](https://user-images.githubusercontent.com/58829855/110961404-118ce200-8350-11eb-80df-00e05cf847f8.png)

Reply:
I think I prefer the version that doesn’t fit perfectly.

Reply:
ok :)